MIGRATION.md

```diff
@@ -1408,3 +1410,30 @@ import {
   <Button variant="primary" size="small" />
 </Modal.Controls>;
 ```
+
+## Tooltip
+
+In version 4, we rename two props of the Tooltip: `containerElement` and `place`.
```

Comment:
I would rephrase this as:

In version 4, we renamed two Tooltip props: `containerElement` is now `as` and `place` is now `placement`.